packages/fxa-settings/src/lib/glean/index.ts

```diff
@@ -323,19 +323,19 @@ const recordEventMetric = (
     case 'password_reset_view':
       passwordReset.view.record();
       break;
-    case 'google_oauth_reg_start':
-      reg.googleOauthRegStart.record();
+    case 'reg_google_oauth_start':
+      reg.googleOauthStart.record();
       break;
-    case 'apple_oauth_reg_start':
-      reg.appleOauthRegStart.record();
+    case 'reg_apple_oauth_start':
+      reg.appleOauthStart.record();
       break;
-    case 'third_party_auth_login_no_pw_view':
+    case 'login_third_party_auth_login_no_pw_view':
```

Comment:
Can we remove the second `login` so it would say `login_third_party_auth_no_pw_view`?